lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -184,7 +184,7 @@ function gutenberg_enqueue_global_styles_custom_css() {
 	remove_action( 'wp_head', 'wp_custom_css_cb', 101 );
 
 	$custom_css  = wp_get_custom_css();
-	$custom_css .= gutenberg_get_global_styles_custom_css();
+	$custom_css .= wp_get_global_styles_custom_css();
```

Comment:
:thinking: same feedback as above. Isn't the point of this PR to make sure we use the gutenberg methods that point to the gutenberg classes? This is a behavior change: as soon as the core function exists, it'll be used instead of gutenberg.